sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    assert (year / week).evalf(15) == Float(52.17745571428571)
```

Comment:
Am 27.03.2015 um 23:40 schrieb Jason K. Moore:
> Regression tests are not only written when bugs are found. You write
> regression tests to catch future bugs regardless if you know about
> the bug or not.

Ah ok. I'd consider that a "correctness test" because the definition of 
"regression test" is another one.

 > It  is especially useful to allow easy refactoring.
> Whether it is in the immediately tested function or something that is
> deep down the stack.

I agree that correctness tests are useful, but I definitely disagree 
that we should add tests for something deep down the stack.

> BTW, I'm annoyed that I'm arguing back and forth with you about a one
> line code change. Is this really that big of a deal to you?

It's about teaching how to write unit tests.
I'm not concerned about the line of code as such.
I'm just annoyed by yet another overspecific test being added. This kind 
of stuff just cost me two weeks of camel shaving (and being unsure 
whether I'm changing semantics or not, because the tests lie to me and 
say the change is breaking things, while the Exprs that now come out are 
*almost* equivalent and I have to find out for each and every one 
whether the differences are something relevant or not).